include/odp/api/spec/timer_types.h

```diff
@@ -102,7 +102,15 @@ typedef struct {
  * Periodic timer capability
  */
 typedef struct {
-	/** Periodic timer pool base frequency in hertz */
+	/**
+	 * Periodic timer pool base frequency in hertz
+	 *
+	 * Base frequency is represented as a fractional number where the fraction part is always
+	 * less than one. In other words, the integer part specifies whole hertz whereas
+	 * the fraction part specifies parts of a hertz (if any). The fraction part does not
+	 * need to be reduced to its lowest terms - e.g. 100.5 Hz may be represented as 100 1/2 Hz,
+	 * 100 5/10 Hz, or 100 Hz with some other equivalent fraction part.
```

Comment:
Could this just be a part of odp_fract_u64_t specification?

Reply:
I think it's better to keep data type documentation generic. For example, in some other use case we might require reduction to the lowest terms, although here we do not require that.